frontend/src/components/TokenSelector/TokenSelector.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { ReactComponent as ETH } from 'assets/icons/ethereum.svg';
+import { ReactComponent as USDC } from 'assets/icons/borrow_usdc.svg';
+import { ReactComponent as STRK } from 'assets/icons/strk.svg';
+import './TokenSelector.css'
+const Tokens = [
+  { id: 'ethOption', component: <ETH />, label: 'ETH' },
+  { id: 'usdcOption', component: <USDC />, label: 'USDC' },
+  { id: 'strkOption', component: <STRK />, label: 'STRK' },
+];
+
+const TokenSelectorDeposit = ({ selectedToken, setSelectedToken }) => {
```

Comment:
it should be called TokenSelector

Reply:
Changed from TokenSelectorDeposit to TokenSelector in the latest pushed